src/Microsoft.ML.FastTree/Microsoft.ML.FastTree.csproj

```diff
@@ -4,7 +4,6 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeInPackage>Microsoft.ML</IncludeInPackage>
 	<DefineConstants>$(DefineConstants);USE_FASTTREENATIVE;NO_STORE;CORECLR</DefineConstants>
-	<EnableDefaultCompileItems>False</EnableDefaultCompileItems>
```

Comment:
If you do this, you need to remove all the `Compile` items below as well.

With doing it, you need to ensure there are no other `.cs` files in this folder that weren't included before.

Reply:
Thank you very much @eerhardt for your help. I deleted the files and ensure no one was encluded before. Just have to wait for the reviews.